public/cli/lib/commands/convert.js

```diff
@@ -155,6 +178,44 @@ Command.run = function(options) {
     });
 };
 
+/**
+ * prepareLogger()
+ * locate the logs directory, and only keep the last X logs.
+ * @param {function} done node style callback(err)
+ * @param {int} numLogsToKeep  how many previous logs to keep.
+ */
+ function prepareLogger( done, numLogsToKeep ) {
```

Comment:
Hmm... can't see where this value gets passed in?

Reply:
I've added in a default value to ```numLogsToKeep```.  

My goal is that this might become a configurable parameter in the app for how many log files to keep.  But that functionality would come later on if we feel like it is worth adding.  Either way, in the mean time, a defaulted parameter works and gives us the flexibility in the future.